docs/search/search_for_content.md

```diff
@@ -51,3 +51,5 @@ Select your filters, then click **Apply** to update the results.
 Click **Clear** to reset and remove all filters.
 
 ![Filtered Search](img/filtered_search.png)
+
+<!--ARCADE EMBED START--><div style="position: relative; padding-bottom: calc(51.27314814814815% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/WuOs9pESmyZLhyyqLqfn?embed&embed_mobile=tab&embed_desktop=inline&show_copy_link=true" title="Search for content" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; color-scheme: light;" ></iframe></div><!--ARCADE EMBED END-->
```

Comment:
I don't have a better idea, so just a remark: the video is at the bottom, after an image which makes it very easy to miss (sometimes it's not even displayed immiedately and needs to load, making it even easier to miss)